pages/_app.js

```diff
@@ -67,18 +67,23 @@ export default class HomePage extends App {
               </Col>
               <Col medium={8} small={4} xsmall={2} large={8}>
                 <nav
+                  aria-label="Menu de Navegação"
                   id="visibleDesktopMenu"
                   className="VisibleMenu  FlexEndPositioning"
                 >
-                  <ul className="NavList">
-                    <li className="DisplayNoneOnSmallScreen">
+                  <ul role="menu" className="NavList">
+                    <li role="menuitem" className="DisplayNoneOnSmallScreen">
                       <a href="/cathocomponents">Catho Components</a>
                     </li>
-                    <li className="DisplayNoneOnSmallScreen">
+                    <li role="menuitem" className="DisplayNoneOnSmallScreen">
                       <a href="/quantum">Quantum</a>
                     </li>
                     <li>
                       <button
+                        aria-haspopup="menu"
+                        aria-controls="hiddenDesktopMenu"
+                        aria-expanded="false"
+                        aria-label="Botão Sanduíche"
```

Comment:
```suggestion
                        aria-label="Menu"
```
Also redundant. I suggest putting “Menu” and in the future we will put something more semantic and dynamic. For example, when the menu is closed the label should be “Open menu” and when open the label should be “Close menu”